src/main/java/evaluation/tournaments/RoundRobinTournament.java

```diff
@@ -2,140 +2,131 @@
 
 import core.AbstractParameters;
 import core.AbstractPlayer;
-import core.interfaces.IStatisticLogger;
 import evaluation.listeners.IGameListener;
-import evaluation.loggers.FileStatsLogger;
 import games.GameType;
 import players.PlayerFactory;
 import players.mcts.BasicMCTSPlayer;
 import players.mcts.MCTSPlayer;
 import players.rmhc.RMHCPlayer;
 import players.simple.OSLAPlayer;
 import players.simple.RandomPlayer;
+import utilities.Pair;
 
 import java.io.FileWriter;
+import java.text.SimpleDateFormat;
 import java.util.*;
 import java.util.stream.Collectors;
 
 import static core.CoreConstants.GameResult;
+import static evaluation.tournaments.RoundRobinTournament.TournamentMode.*;
+import static java.util.stream.Collectors.toList;
 import static utilities.Utils.getArg;
 
 
 public class RoundRobinTournament extends AbstractTournament {
```

Comment:
If this mashed with GameReport and GameReport is gone... can we have the main class to run things in the evaluation package (rather than evaluation.tournaments) and named so that it's clear it's *the* main thing to run for game/agent evaluation?